ngrok/src/tunnel_ext.rs

```diff
@@ -60,6 +62,12 @@ pub trait TunnelExt: Tunnel {
     async fn forward_http(&mut self, addr: impl ToSocketAddrs + Send) -> Result<(), io::Error> {
         forward_conns(self, addr, |e, c| drop(serve_gateway_error(e, c))).await
     }
+
+    /// Forward incoming tunnel connections to the provided Unix socket path.
+    #[instrument(level = "debug", skip_all, fields(local_addrs))]
```

Comment:
You'll need to replace `local_conns` with `path` here to get the field recorded right.

Reply:
Updated to path